lib/src/actions/actions.dart

```diff
@@ -2066,6 +2066,21 @@ abstract class ShowGridCoordinatesSideViewSet
       _$showGridCoordinatesSideViewSetSerializer;
 }
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////
+// add option to not save DNADesign in localStorage on every edit
+
+abstract class SaveDNAInLocalStorageSet
```

Comment:
Can we replace "DNA" with "DNADesign"? In other words change the Action to `SaveDNADesignInLocalStorageSet`, and the same for the field wherever it appears.

Just calling it "DNA" makes it sound like it's only the DNA sequence being saved, not the whole design.